backend/kernelCI_app/views/treeCommitsHistory.py

```diff
@@ -262,6 +283,7 @@ def get(self, request, commit_hash):
                 status=status.HTTP_400_BAD_REQUEST,
             )
 
+        self._process_time_range(request)
```

Comment:
I think you could process the time range only if it isn't None, otherwise it will throw an Exception every time you access this endpoint without the start/end time (which is what happens in any treeDetails page at the moment)